### ISSUE TITLE
check_ncpa.py: Provide more detailed informations in case of connection errors, http errors and errors thrown by the agent

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#573]
check_ncpy.py version:
```
20:17:51 [nagios@localhost]:~/tmp$ ./check_ncpa.py --version
check_ncpa.py, Version 1.1.6
```

Current behaviour in case of a connection error (host doesnt exist):
```
20:17:54 [nagios@localhost]:~/tmp$ ./check_ncpa.py -H 10.0.0.7 -t mytoken -M '/disk/logical/|tmp'
UNKNOWN: Error occurred while running the plugin. Use the verbose flag for more details.
```

Plugin call with verbose mode in case of a connection error:
```
20:18:02 [nagios@localhost]:~/tmp$ ./check_ncpa.py -H 10.0.0.7 -t mytoken -M '/disk/logical/|tmp' -v
Connecting to: https://10.0.0.7:5693/api/disk/logical/%7Ctmp/?token=mytoken&check=1
An error occurred:<urlopen error [Errno 113] No route to host>
```

Current behaviour in case of a http error:
```
20:19:38 [nagios@localhost]:~/tmp$ ./check_ncpa.py -H localhost -P 443 -t mytoken -M '/disk/logical/|tmp' 
UNKNOWN: Error occurred while running the plugin. Use the verbose flag for more details.
```

Plugin call with verbose mode in case of a connection error:
```
20:20:30 [nagios@localhost]:~/tmp$ ./check_ncpa.py -H localhost -P 443 -t mytoken -M '/disk/logical/|tmp'  -v
Connecting to: https://localhost:443/api/disk/logical/%7Ctmp/?token=mytoken&check=1
An error occurred:HTTP Error 404: Not Found
```

I would like to see the informations provided in verbose mode in the normal plugin output. The message: ```UNKNOWN: Error occurred while running the plugin. Use the verbose flag for more details.``` is meaningless. I will provide a pull request for that.

**Comment by ccztux** [comment]
With commit: 87f523c2698aa9794e5e25033eae8a6534c92c80 errors will displayed in output instead of the generic message: ```UNKNOWN: Error occurred while running the plugin. Use the verbose flag for more details.```

Old behaviour in case of wrong token:
```
18:50:23 [nagios@localhost]:~# ./check_ncpa.py -H 10.0.0.73 -t ytoken -M '/disk/logical/|ttmp' 
UNKNOWN: Error occurred while running the plugin. Use the verbose flag for more details.

18:52:12 [nagios@localhost]:~# ./check_ncpa.py -H 10.0.0.73 -t ytoken -M '/disk/logical/|ttmp' -v
Connecting to: https://10.0.0.73:5693/api/disk/logical/%7Cttmp/?token=ytoken&check=1
File returned contained:
{
  "error": "Incorrect credentials given."
}
An error occurred:
'stdout'
```


New behaviour in case of wrong token:
```
18:50:34 [nagios@localhost]:~/Dokumente/git-repos/ncpa/client$ ./check_ncpa.py -H 10.0.0.73 -t ytoken -M '/disk/logical/|ttmp' 
Incorrect credentials given.
```

**Comment by jomann09** [comment]
I've merged this into maint and will be released as part of a 1.2.0 version of check_ncpa. Thanks a lot for working on this!
